src/react/2-intro-to-jsx.md

```diff
@@ -9,13 +9,13 @@
 
 Before we talk about JSX, let's discuss how user interfaces are created in React.
 
-At its core, React is a JavaScript library that helps create reusable front-end components. One of the unique features of React is that it allows you to describe the way your components look using only JavaScript.
+At its core, React is a JavaScript library that helps create reusable front-end views. One of the unique features of React is that it allows you to describe the way your components look using only JavaScript.
```

Comment:
I'm not sure if "Views" is a more widely known term than "Components". The less new terminology we have to introduce, the better.

Reply:
Agreed, I changed it back. It came out of a convo I had with Justin but I think components makes more sense here.